Gruntfile.js

```diff
@@ -39,13 +39,21 @@ module.exports = function (grunt) {
 		nodeunit: {
 			all: 'test/node.js'
 		},
+		mochaTest: {
```

Comment:
I'm not sure I understand the problem. Are you saying you want mocha to run without running babel? If we do that, then we can't test es6 imports.